src/sagas/bucket.js

```diff
@@ -37,10 +37,13 @@ export function* createBucket(getState, action) {
 
 export function* updateBucket(getState, action) {
   const {bid, bucketData} = action;
+  const {bucket: currentBucket} = getState();
+  const {last_modified} = currentBucket;
+  const savedBucket = {...bucketData, last_modified};
```

Comment:
Nit: this gives the false impression the bucket is already saved; `updatedBucket`?